ui/src/app/components/direct-connection/direct-connection.component.html

```diff
@@ -46,6 +46,16 @@ <h3 class="primary-header">Connection Detail</h3>
         <input matInput name="dbname" type="text" formControlName="dbName" />
       </mat-form-field>
       <br />
+      <h3 class="primary-header">Spanner Dialect</h3>
+      <mat-form-field class="full-width" appearance="outline">
+        <mat-label>Spanner Dialect</mat-label>
+        <mat-select matSelect name="dialect" formControlName="dialect" appearance="outline">
+          <mat-option *ngFor="let element of dialect" [value]="element.value">
+            {{ element.displayName }}
```

Comment:
Should dialect be part of the `direct-connection` page? I don't think its relevant for the user to think of what dialect of Spanner they want to migrate to at this stage, what do you think?

In my opinion, dialect information should be taken as input in the `workspace`? (Not quite sure what technical challenges it will present).

Reply:
Based on the discussion keeping it in direct-connection page